src/Utils/QueryAnalyzer.php

```diff
@@ -483,46 +512,42 @@ public function track_nodes( $model ) {
 		return $model;
 	}
 
-
 	/**
-	 * Return headers
-	 *
-	 * @param array $headers The array of headers being returned
+	 * Returns graphql keys for use in debugging and headers.
 	 *
 	 * @return array
 	 */
-	public function get_headers( array $headers = [] ): array {
+	public function get_graphql_keys() {
 
-		$keys = [];
+		if ( ! empty( $this->graphql_keys ) ) {
+			return $this->graphql_keys;
+		}
+
+		$keys        = [];
+		$return_keys = '';
 
 		if ( $this->get_query_id() ) {
 			$keys[] = $this->get_query_id();
 		}
 
 		if ( ! empty( $this->get_root_operation() ) ) {
-			$headers['X-GraphQL-Operation-Type'] = $this->get_root_operation();
```

Comment:
no longer setting this header. intentional?

Reply:
Ya, I was trying to reduce the headers. This one is probably not a big deal to stay, but the `X-GraphQL-Nodes` is the main culprit of the `X-GraphQL-Keys` header being too long. Def don't want it exposed 2x.